shaders/src/getters.fs

```diff
@@ -1,13 +1,11 @@
 //------------------------------------------------------------------------------
-// Instance access
+// Uniforms access
 //------------------------------------------------------------------------------
 
-#if defined(MATERIAL_HAS_INSTANCES)
-/** @public-api */
-int getInstanceIndex() {
-    return instance_index;
+/** sort-of public */
+float getObjectUserData() {
+    return gObjectUniforms.userData;
```

Comment:
Note: we don't use the g prefix for other globals. But we used _, so like object_uniforms. Doesn't matter much though.

Reply:
done